src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/ReasonableDefaults.drl

```diff
@@ -41,3 +41,14 @@ rule "Fill 'Insights' field with reasonable default"
     then
         workloadInventoryReport.setInsightsEnabled(WorkloadInventoryReportModel.INSIGHTS_ENABLED_DEFAULT_VALUE);
 end
+
+rule "Fill 'osName' field with reasonable default"
+       when
+           workloadInventoryReport : WorkloadInventoryReportModel(
+               (osName == null && osDescription == null) ||
```

Comment:
Only as a style suggestion, this 8 comparators line could be replaced by `StringUtils.isAllBlank(osName,odDescription)`

Reply:
Done